redfish-core/lib/pcie_slots.hpp

```diff
@@ -325,7 +327,8 @@ inline void
         const std::string& fabricAdapterPath = fabricAdapterPaths.front();
         nlohmann::json& slot = asyncResp->res.jsonValue["Slots"][index];
 
-        slot["Links"]["Oem"]["@odata.type"] = "#OemPCIeSlots.Oem";
+        slot["Links"]["Oem"]["odata.type"] = "#OemPCIeSlots.Oem";
```

Comment:
This looks wrong.. isn't it always '@odata.type'

Reply:
Thanks for finding out the bug.
Fixed.